examples/simpleapp/mc_sc_workflow_example.md

```diff
@@ -85,16 +85,27 @@ For retrieving other Schnorr keys repeat this command.
 
 **Step 6: Generate data for certificate proof creation and verification**
 
-Creation of Sidechains requires data for proving backward transfer operations, included in certificates. That data could be generated by next command:
+Creation of Sidechains requires data for proving backward transfer operations, included in certificates.
+*If circuit without key rotation, that data could be generated by next command:*
 
 `generateCertProofInfo {"signersPublicKeys": [pk1, pk2, ...], "threshold": 5, "verificationKeyPath": "/tmp/sidechainapp/cert_marlin_snark_vk", "provingKeyPath": "/tmp/sidechainapp/cert_marlin_snark_pk", "isCSWEnabled": true}`
 
-Note: 
+Note:
 - `signersPublicKeys` - list of Schnorr public keys of certificate Signers generated on step 5;
 - `threshold` - the minimum set of the participants required for a valid proof creation;
-- `isCSWEnabled` - Its value should be true if the Ceased Sidechain Withdrawal is enabled, false otherwise.
+- `isCSWEnabled` - Its value should be true if the Ceased Sidechain Withdrawal is enabled, false otherwise. For non ceasing sidechains it's always false.
+
+*If circuit with key rotation:*
 
+`generateCertWithKeyRotationProofInfo {"signersPublicKeys": [signerPk1, signerPk2, ...], "mastersPublicKeys": [masterPk1, masterPk2, ...], "threshold": 5, "verificationKeyPath": "/tmp/sidechainapp/cert_marlin_snark_vk", "provingKeyPath": "/tmp/sidechainapp/cert_marlin_snark_pk", "isCSWEnabled": true}`
```

Comment:
Please change isCSWEnabled to false cause it's not supported with keyRotationCircuit.

Reply:
Done